doc/ins_attest_utxo.md

```diff
@@ -63,6 +63,8 @@ Upon receiving last txChunk (as determined by the transaction parsing) the respo
 
 Note on HMAC: We use `HMAC_SHA256(key=32 byte session key, message=(txHash,outNum,amount))` to compute the signature digest and return first 16 bytes of it. This should be resilient enough to birthday paradox and other attacks as the key is just per session and we Ledger's signing speed is slow. ❓(IOHK): Is this enough? 
 
+**IOHK vincenthz/nicolas**: we're not quite sure what is this supposed to protect from, and if that's useful for anything until we have the attested data on the ledger validated directly anyway.
```

Comment:
@vincenthz @NicolasDP The idea is that attestUtxo will *return* attestation and this attestation is needed in the subsequent `signTx`. The rationale behind this is that Ledger might not be able to store attestation for all UTxOs and so we would stream them one-by-one in signTx call)

Reply:
apologies if this is mentioned somewhere and I've missed it, but I don't understand what's those attestation are for in the first place, nor what they are supposed/trying to protect from.